### ISSUE TITLE
configure.ac doesn't detect no gettext

### ISSUE THREAD

**Issue by rossburton** [util-linux/util-linux#1372]
If you don't have any gettext on your build machine (say, because it's a fresh minimal install), `configure` still succeeds:

```
checking whether NLS is requested... yes
checking for msgfmt... no
checking for gmsgfmt... :
checking for xgettext... no
checking for msgmerge... no
[...]
 Type 'make' or 'make <utilname>' to compile.
```

But the build doesn't:

```
make[3]: Entering directory '/home/ross/Code/util-linux/po'
: --update --no-wrap --previous --lang=ca ca.po util-linux.pot
rm -f ca.gmo && : -c --statistics --verbose -o ca.gmo ca.po
mv: cannot stat 't-ca.gmo': No such file or directory
```

**Comment by karelzak** [comment]
Yes, I know (see commit 67735475db2e6da5668f4c21a8736bd3539737aa), but the gettext (and friends) should be optional as it's unnecessary when you compile sources from the tarball (in this case we need "make install" from po/Makefile only), and we do not want to ignore the entire po/ when something is missing.

The question is how we can fix it as po/Makefile is generated by autopoint (called from ./autogen.sh) ...

**Comment by rffontenelle** [comment]
@rossburton configure's `--disable-nls` flag should avoid this issue.

**Comment by rffontenelle** [comment]
> @rffontenelle the point is I wanted NLS to work, but I had missing build dependencies.

Oh, I see. I thought it was the opposite since normally gettext is installed, or people install it when desired.

> It should error nicely at configure time instead of aborting mysteriously mid-build.

Actually, missing gettext should not give error by default. 

As karelzak mentioned, Gettext is an optional dependency because it  is *not* indispensable for the proper use (in English) of most software. It would be a hard dependency if this was a localization-related software (e.g. po4a).

If your system doesn't enforce gettext as installed software (e.g. some container images does this to save disk space), I suggest installing gettext before building software which you want to have to NLS (native language support).

But, indeed, util-linux should not pass ./configure disabling NLS, and then try to build locales without gettext. That's a bug.

**Comment by mator** [comment]
i can't reproduce with current git version of util-linux
